native/desktop-windows/src/win32/renderer_angle.rs

```diff
@@ -0,0 +1,190 @@
+        }
+
+        self.surface = unsafe {
+            self.egl_instance
+                .create_window_surface(self.display, self.surface_config, self.window.0 as _, Some(&surface_attribs))
+        }?;
+
+        self.egl_instance
+            .make_current(self.display, Some(self.surface), Some(self.surface), Some(self.context))?;
+        self.egl_instance.swap_interval(self.display, 1)?;
+
+        Ok(())
+    }
+
+    pub fn draw(&self, wait_for_vsync: bool, draw_fun: AngleDeviceDrawFun) -> Result<()> {
+        self.egl_instance
+            .make_current(self.display, Some(self.surface), Some(self.surface), Some(self.context))?;
+
+        draw_fun();
+
+        self.egl_instance.swap_interval(self.display, if wait_for_vsync { 1 } else { 0 })?;
+        self.egl_instance.swap_buffers(self.display, self.surface)?;
+
+        Ok(())
+    }
+
+    #[inline]
+    pub fn get_proc_address(&self, procname: &str) -> Option<extern "system" fn()> {
+        self.egl_instance.get_proc_address(procname)
+    }
+}
+
+impl Drop for AngleDevice {
+    fn drop(&mut self) {
+        let _ = self.egl_instance.make_current(self.display, None, None, None);
+        if self.context.as_ptr() != egl::NO_CONTEXT {
+            let _ = self.egl_instance.destroy_context(self.display, self.context);
+        }
+        if self.surface.as_ptr() != egl::NO_SURFACE {
+            let _ = self.egl_instance.destroy_surface(self.display, self.surface);
+        }
+        if self.display.as_ptr() != egl::NO_DISPLAY {
+            let _ = self.egl_instance.terminate(self.display);
+        }
+    }
+}
+
+fn get_angle_platform_display(egl_instance: &EglInstance, hdc: &HDC) -> Result<egl::Display> {
+    type GetPlatformDisplayEXTFn =
+        extern "C" fn(platform: egl::Enum, native_display: *mut std::ffi::c_void, attrib_list: *const egl::Int) -> egl::EGLDisplay;
+
+    const EGL_PLATFORM_ANGLE_ANGLE: egl::Int = 0x3202;
+    const EGL_PLATFORM_ANGLE_TYPE_ANGLE: egl::Int = 0x3203;
+
+    //const EGL_PLATFORM_ANGLE_TYPE_D3D9_ANGLE: egl::Int = 0x3207;
+    const EGL_PLATFORM_ANGLE_TYPE_D3D11_ANGLE: egl::Int = 0x3208;
+
+    let fun: GetPlatformDisplayEXTFn = egl_instance
+        .get_proc_address("eglGetPlatformDisplayEXT")
+        .ok_or_else(|| anyhow!("Could not load the eglGetPlatformDisplayEXT function."))
+        .map(|f| unsafe { core::mem::transmute(f) })?;
+
+    #[rustfmt::skip]
+    let display_attribs = [
+        EGL_PLATFORM_ANGLE_TYPE_ANGLE, EGL_PLATFORM_ANGLE_TYPE_D3D11_ANGLE,
+        egl::NONE, egl::NONE,
+    ];
+
+    match fun(EGL_PLATFORM_ANGLE_ANGLE as _, hdc.0, display_attribs.as_ptr()) {
+        egl::NO_DISPLAY => Err(egl_instance
+            .get_error()
+            .map_or_else(|| anyhow!("Could not get ANGLE platform display."), |err| err.into())),
+        display => Ok(unsafe { egl::Display::from_ptr(display) }),
+    }
+}
+
+fn load_angle_libraries() -> Result<libloading::Library> {
```

Comment:
Another option would be to explicitly pass the path to libraries from Kotlin, we already have it `KotlinDesktopToolkit.init()`